### ISSUE TITLE
Three- Finger Gesture is not working with Voice Over on Webview Carousel. 

### ISSUE THREAD

**Issue by A11yTeam** [ionic-team/ionic-v1#355]
PLatform : ios 11.2 Webview - Carousel 

In iOS 11.2 , VoiceOver enabled, users are unable to three-finger swipe to move to next carousel tile on Webview. 

Steps to reproduce: Webview
In iOS 11.2 enable VoiceOver in the Accessibility menu
Open the test app
Three-finger swipe left or right to move to next carousel tile.

Expected behaviour:
Three-finger swiping with VoiceOver enabled must act the same as single-finger swiping with VO disabled.

Please advice on this issues.

Thanks.
